.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-pexels
\ No newline at end of file
+pexels
+config.json
```

Comment:
@Tokki0401 good catch on config.json. Odd because it is in my .gitignore. I'll investigate this further